src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,31 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        Reader reader = new Reader();
+        Writer writer = new Writer();
```

Comment:
The 'Reader' and 'Writer' classes are not defined in the provided code. Ensure that these classes exist and follow Java naming conventions for classes, which suggest they should be named with nouns and in CamelCase, such as 'CsvReader' and 'CsvWriter'.